flume-ng-core/src/test/java/org/apache/flume/source/TestAvroSource.java

```diff
@@ -97,20 +96,19 @@ public void testLifecycle() throws InterruptedException {
     boolean bound = false;
 
     for (int i = 0; i < 100 && !bound; i++) {
-      try {
-        Context context = new Context();
-
-        context.put("port", String.valueOf(selectedPort = 41414 + i));
-        context.put("bind", "0.0.0.0");
 
-        Configurables.configure(source, context);
+      Context context = new Context();
 
+      context.put("port", String.valueOf(selectedPort = 41414 + i));
+      context.put("bind", "0.0.0.0");
+      // Invalid configuration may throw a FlumeException which has to be expected in the callers
```

Comment:
Could you please explain what this comment means? Or is it related to the changes you made, i.e. removing the configuration part from the `try-catch` block?